README.rst

```diff
@@ -27,7 +27,9 @@ PyGMT
 .. image:: https://img.shields.io/discourse/status?label=forum&server=https%3A%2F%2Fforum.generic-mapping-tools.org%2F&style=flat-square
     :alt: Discourse forum
     :target: https://forum.generic-mapping-tools.org
-
+.. image:: https://zenodo.org/badge/123456789.svg
+    :alt: Digital Object Identifier for the Zenodo archive
+    :target: https://zenodo.org/badge/latestdoi/123456789
```

Comment:
```suggestion
.. image:: https://zenodo.org/badge/123456789.svg
    :alt: Digital Object Identifier for the Zenodo archive
    :target: https://zenodo.org/badge/latestdoi/123456789
```
TODO insert correct Concept DOI here.

Reply:
Ok, I *think* I've managed to reserve a DOI on Zenodo. Seems like we'll need to make a release and then upload the repository zip/tarball before it becomes active though.

![Zenodo DOI reserved image](https://user-images.githubusercontent.com/23487320/80853424-e98b1780-8c84-11ea-9b72-0916dc876ace.png)

Reply:
As far as I know, zenodo provides two types of DOI. Each release has its own doi, and the whole project has a permanent doi which is always resolved to the lastest release. 

For example, https://doi.org/10.5281/zenodo.1068342 is the permanent doi for one of my project, and https://doi.org/10.5281/zenodo.3695076 is the doi for the v0.6.6 release.

Perhaps we should use the permanent doi in the README.rst and the release-specific doi in the changelog.

Reply:
Yes, but right now it's in a draft state and it's only showing me one DOI... I could upload a dummy zipfile and see if that gives me two DOIs?

Reply:
This is currently what I've filled in on Zenodo. I've listed everyone in our AUTHORS.md file, and Paul on the author list, and also added the NSF funding information.

![Zenodo details](https://user-images.githubusercontent.com/23487320/80854620-b4cf8e00-8c8d-11ea-94f4-2491e5d46d32.png)